src/Symfony/Bundle/Resources/config/routing/api.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Routing\Loader\Configurator;
+
+return function (RoutingConfigurator $routes) {
+    $routes->add('api_entrypoint', '/{index}.{_format}')
```

Comment:
I thought we could put `index` in the path, since there is a requirements `index=index`, but it turns out that index is optional. This route matches `/`